### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/layout/breadcrumb.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#700] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/layout/breadcrumb.html

vuepress-theme-hope 添加了开箱即用的路径导航支持，你可以通过 breadcrumb 字段控制它 无需任何额外配置，一个和主题色相符合的路径导航会显示在页面内容最上方，帮助阅读者理解文档结构。 你也可以通过 breadcrumbIcon 字段

https://theme-hope.vuejs.press/zh/guide/layout/breadcrumb.html

**Comment by 006hjy** [comment]
如果路径导航可以和目录一样自定义根路径就好了

**Reply by Mister-Hope** [reply]
路径导航本来就是显示网站结构的，虽然说自定义根路径可能会在很深层级的文档中，对用户快速导航起到一定帮助，但这不是breadcrumb 本身的意义。
